examples/tv-casting-app/linux/CastingUtils.cpp

```diff
@@ -100,7 +100,8 @@ void InitCommissioningFlow(intptr_t commandArg)
             CastingServer::GetInstance()->GetDiscoveredCommissioner(i, associatedConnectableVideoPlayer);
         if (commissioner != nullptr)
         {
-            ChipLogProgress(AppServer, "Discovered Commissioner #%d", commissionerCount++);
+            ChipLogProgress(AppServer, "Discovered Commissioner #%d", commissionerCount);
+            commissionerCount++;
```

Comment:
do we subtract this anywhere? or is it purely incremental here?

Reply:
This one is just incremental, because in this version of the casting APIs on Linux, there is no API to remove a previously discovered commissioner. Only incrementing is OK also because the enclosing function InitCommissioningFlow is just called once.